admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/ZoneServiceImpl.java

```diff
@@ -95,6 +96,7 @@
 	 * io.mosip.kernel.masterdata.service.ZoneService#getUserLeafZone(java.lang.
 	 * String)
 	 */
+	@Cacheable(value = "zones", key = "'zone'.concat('-').concat(#langCode)")

```

Comment:
There are other methods that are missing? Seems like in each file we have missed some of the api's